sources/assets/exegol/start.sh

```diff
@@ -2,38 +2,38 @@
 
 function shell_logging() {
     # First parameter is the method to use for shell logging (default to script)
-    METHOD=$1
+    local method=$1
     # The second parameter is the shell command to use for the user
-    USER_SHELL=$2
+    local user_shell=$2
     # The third enable compression at the end of the session
-    COMPRESS=$3
+    local compress=$3
 
-    # Logging shell using $METHOD and spawn a $USER_SHELL shell
+    # Logging shell using $method and spawn a $user_shell shell
 
     umask 007
     mkdir -p /workspace/logs/
-    FILELOG="/workspace/logs/$(date +%d-%m-%Y_%H-%M-%S)_shell.${METHOD}"
+    local filelog="/workspace/logs/$(date +%d-%m-%Y_%H-%M-%S)_shell.${method}"
```

Comment:
This needs to be declared and assigned separately, like so
```bash
local filelog
filelog="/workspace/logs/$(date +%d-%m-%Y_%H-%M-%S)_shell.${method}"
```

Reply:
Done